metaserver/src/test/scala/tests/compiler/CompletionsTest.scala

```diff
@@ -169,18 +172,53 @@ object CompletionsTest extends CompilerSuite {
   check(
     "package",
     """
-      | import scala.collect<<>>
+      |import scala.collect<<>>
     """.stripMargin,
     s"""
        |{
        |  "isIncomplete" : false,
        |  "items" : [ {
        |    "label" : "collection",
        |    "kind" : ${CompletionItemKind.Module.value},
-       |    "detail" : ""
+       |    "detail" : "",
+       |    "sortText" : "00000"
```

Comment:
Feels quite boilerplatey to copy-paste this for all unit tests against a single completion, can we create a more targeted test to only check against label and kind?

Reply:
done!